TYPO3.Neos/Resources/Private/Partials/Module/Shared/NewElectronicAddress.html

```diff
@@ -2,7 +2,7 @@
 <f:form action="createElectronicAddress" name="electronicAddress" object="{electronicAddress}" arguments="{user: user}">
 	<div class="neos-row-fluid neos-module-container">
 		<fieldset class="neos-span5">
-			<legend>{neos:backend.translate(id: 'userSettings.addNewElectronicAddress', value: 'Add new electronic address')}</legend>
+			<legend>{neos:backend.translate(id: 'userSettings.electronicAddresses.addElectronicAddress', value: 'Add new electronic address', source: 'Modules')}</legend>
```

Comment:
maybe this should be moved so it doesn't reference userSettings, since it's a shared template.. also there's a ``electronicAddresses`` to put it in now..